airflow/utils/log/es_task_handler.py

```diff
@@ -28,6 +32,43 @@
 from airflow.utils.log.logging_mixin import LoggingMixin
 
 
+class ParentStdout():
+    """
+    Keep track of the ParentStdout stdout context in child process
+    """
+    def __init__(self):
+        self.closed = False
+
+    def write(self, string):
+        sys.__stdout__.write(string)
+
+    def close(self):
+        self.closed = True
+
+
+class JsonFormatter(logging.Formatter):
+    """
+    Custom formatter to allow for fields to be captured in JSON format.
+    Fields are added via the RECORD_LABELS list.
+    TODO: Move RECORD_LABELS into configs/log_config.py
```

Comment:
should we finish it rather than set it `todo`?

Reply:
agreed- this was an old TODO, removed! thanks for the catch